nomad/job_endpoint.go

```diff
@@ -441,8 +441,10 @@ func (j *Job) Revert(args *structs.JobRevertRequest, reply *structs.JobRegisterR
 	}
 
 	// Build the register request
+	revJob := jobV.Copy()
+	revJob.VaultToken = args.VaultToken
```

Comment:
Let's comment this.

```suggestion
	// Use Vault Token from revert request to perform registration of reverted job.
	revJob.VaultToken = args.VaultToken
```

...or something like that?